file-formats/gsmp/type/zif_aff_gsmp_v1.intf.abap

```diff
@@ -101,6 +101,10 @@ INTERFACE zif_aff_gsmp_v1
       "! No execution. Only model. Data from external.
       none        TYPE ty_execution_mode VALUE 'N',
 
+      "! <p class="shorttext">Job</p>
+      "! Job.
+      job        TYPE ty_execution_mode VALUE 'J',
+
```

Comment:
Minor: Would it make sense to move `job` before `none` so that `none` is the last entry?

Reply:
Wird heute Mittag korrigiert

Reply:
done ...